Two `TestBatch` failures in macOS x86-64 Accelerate wheel build jobs

The macOS x86-64 Accelerate wheel builds are all showing these two test failures (e.g., see [this log](https://github.com/scipy/scipy/actions/runs/12985610874/job/36210824465)):

```
  ____________________ TestBatch.test_solve[complex64-bdim2] _____________________
  ../venv-test-x86_64/lib/python3.13/site-packages/scipy/linalg/tests/test_batch.py:413: in test_solve
      assert_allclose(A @ x - b, 0, atol=1e-6)
  E   AssertionError: 
  E   Not equal to tolerance rtol=1e-07, atol=1e-06
  E   
  E   Mismatched elements: 1 / 120 (0.833%)
  E   Max absolute difference among violations: 1.06467728e-06
  E   Max relative difference among violations: inf
  E    ACTUAL: array([[[[-3.725290e-09+2.980232e-08j,  1.490116e-07+5.960464e-08j,
  E             -5.960464e-08-5.960464e-08j, -8.940697e-08+1.192093e-07j],
  E            [ 5.960464e-08+0.000000e+00j,  1.192093e-07-4.470348e-08j,...
  E    DESIRED: array(0)
          A          = array([[[[0.8561369 +0.05969837j, 0.37626302+0.36519146j,
            0.6539293 +0.40403795j, 0.16993557+0.79370755j,
    ...1648j,
            0.50310206+0.6898472j , 0.6573576 +0.9962738j ,
            0.2530085 +0.86227924j]]]], dtype=complex64)
          b          = array([[[[0.04875669+0.31144133j, 0.45005864+0.22624582j,
            0.797893  +0.5535813j , 0.41498896+0.6063427j ],
   ...94375426j, 0.05030875+0.60288817j,
            0.673049  +0.02498303j, 0.7586571 +0.54190135j]]]],
        dtype=complex64)
          bdim       = (2, 3, 5, 4)
          dtype      = <class 'numpy.complex64'>
          rng        = Generator(PCG64) at 0x1304A33E0
          self       = <scipy.linalg.tests.test_batch.TestBatch object at 0x11dc1cdc0>
          x          = array([[[[-0.1252879 +1.7789268e-03j,  0.50262445+8.4138566e-01j,
             0.6212484 -4.7690693e-01j,  0.82990605-3....0.14644781+6.8131542e-01j,
            -0.91509104+8.1870025e-01j,  0.05595069-2.2421245e-01j]]]],
        dtype=complex64)
  ___________________ TestBatch.test_lu_solve[complex64-bdim2] ___________________
  ../venv-test-x86_64/lib/python3.13/site-packages/scipy/linalg/tests/test_batch.py:426: in test_lu_solve
      assert_allclose(A @ x - b, 0, atol=1e-6)
  E   AssertionError: 
  E   Not equal to tolerance rtol=1e-07, atol=1e-06
  E   
  E   Mismatched elements: 1 / 120 (0.833%)
  E   Max absolute difference among violations: 1.06467728e-06
  E   Max relative difference among violations: inf
  E    ACTUAL: array([[[[-3.725290e-09+2.980232e-08j,  1.490116e-07+5.960464e-08j,
  E             -5.960464e-08-5.960464e-08j, -8.940697e-08+1.192093e-07j],
  E            [ 5.960464e-08+0.000000e+00j,  1.192093e-07-4.470348e-08j,...
  E    DESIRED: array(0)
          A          = array([[[[0.8561369 +0.05969837j, 0.37626302+0.36519146j,
            0.6539293 +0.40403795j, 0.16993557+0.79370755j,
    ...1648j,
            0.50310206+0.6898472j , 0.6573576 +0.9962738j ,
            0.2530085 +0.86227924j]]]], dtype=complex64)
          b          = array([[[[0.04875669+0.31144133j, 0.45005864+0.22624582j,
            0.797893  +0.5535813j , 0.41498896+0.6063427j ],
   ...94375426j, 0.05030875+0.60288817j,
            0.673049  +0.02498303j, 0.7586571 +0.54190135j]]]],
        dtype=complex64)
          bdim       = (2, 3, 5, 4)
          dtype      = <class 'numpy.complex64'>
          lu_and_piv = [array([[[[ 0.78804415+0.9227373j ,  0.69236445+0.7835633j ,
             0.31624767+0.2628439j ,  0.9063425 +0.35367382...,
          [2, 2, 2, 3, 4]],
  
         [[2, 2, 4, 4, 4],
          [0, 2, 3, 3, 4],
          [2, 1, 4, 3, 4]]], dtype=int32)]
          rng        = Generator(PCG64) at 0x1304A3E60
          self       = <scipy.linalg.tests.test_batch.TestBatch object at 0x11dc1d590>
          x          = array([[[[-0.1252879 +1.7789268e-03j,  0.50262445+8.4138566e-01j,
             0.6212484 -4.7690693e-01j,  0.82990605-3....0.14644781+6.8131542e-01j,
            -0.91509104+8.1870025e-01j,  0.05595069-2.2421245e-01j]]]],
        dtype=complex64)
```

Cc @mdhaber

--- a/scipy/linalg/tests/test_batch.py
+++ b/scipy/linalg/tests/test_batch.py
@@ -410,7 +410,7 @@ class TestBatch:
         if len(bdim) == 1:
             x = x[..., np.newaxis]
             b = b[..., np.newaxis]
-        assert_allclose(A @ x - b, 0, atol=1e-6)
+        assert_allclose(A @ x - b, 0, atol=1.5e-6)
         assert_allclose(x, np.linalg.solve(A, b), atol=2e-6)
 
     @pytest.mark.parametrize('bdim', [(5,), (5, 4), (2, 3, 5, 4)])
@@ -423,7 +423,7 @@ class TestBatch:
         if len(bdim) == 1:
             x = x[..., np.newaxis]
             b = b[..., np.newaxis]
-        assert_allclose(A @ x - b, 0, atol=1e-6)
+        assert_allclose(A @ x - b, 0, atol=1.5e-6)
         assert_allclose(x, np.linalg.solve(A, b), atol=2e-6)
 
     @pytest.mark.parametrize('l_and_u', [(1, 1), ([2, 1, 0], [0, 1 , 2])])

